### ISSUE TITLE
PurgeCSS

### ISSUE THREAD

**Issue by danielehrhardt** [notiz-dev/ngx-tailwind#10]
Could you maybe add PurgeCSS?

**Comment by marcjulian** [comment]
@danielehrhardt `tailwindcss` requires `@fullhuman/postcss-purgecss` as a dependency, which on the other hand requires `purgecss`. What is your benefit of adding PurgeCSS to the schematics?

**Comment by danielehrhardt** [comment]
I was able to size down the generated `style.css` File from 5MB to 10KB.

![image](https://user-images.githubusercontent.com/19570043/100623247-2f46b600-3322-11eb-94ea-c04caaf983f6.png)

**Comment by marcjulian** [comment]
Right you can also move those `purgecss` rules into the `tailwind.config.js`

```js
module.exports = {
  purge: [
    './src/**/*.html',
    './src/**/*.ts'
  ],
 dark: false,
  theme: {
    extend: { }
  },
  variants: {},
  plugins: []
};
```

You are not limited to have `purge` as array it also can be an object with advanced syntax like you did in the `webpack.config.js`

https://tailwindcss.com/docs/optimizing-for-production#removing-unused-css

**Comment by marcjulian** [comment]
Here is an example from the blog.tailwindcss.com project

https://github.com/tailwindlabs/blog.tailwindcss.com/blob/364b3713862b6d309aff79505776a9948ad26620/tailwind.config.js#L5-L27

**Comment by danielehrhardt** [comment]
Okay, thank you. Maybe you could add this to the Readme File. 
You know developers are lazy. 🙈

**Comment by marcjulian** [comment]
This schematics tries to make the integration of Tailwind CSS as easy as possible and all Tailwind configs are supposed to continue working. But you might be right that a note in the Readme file wouldn't hurt. Do you mind creating a PR adding the note to the readme? 😄

**Comment by garygrossgarten** [comment]
Why not add a default purge config in tailwind config?

**Comment by marcjulian** [comment]
Default options like this

```js
...
purge: ['./src/**/*.html', './src/**/*.ts'],
...
```

**Comment by marcjulian** [comment]
I am reopening as we are going to add default options to `tailwind.config.js` for purging in production.

**Comment by marcjulian** [comment]
Closed by a6e309b

**Comment by marcjulian** [comment]
Released in [v1.1.0](https://github.com/notiz-dev/ngx-tailwind/releases/tag/v1.1.0). Please give it a try!
